### ISSUE TITLE
Suggestion: support Shoulder Surfing mod

### ISSUE THREAD

**Issue by SepSol** [Snownee/Nimble#4]
Please.

**Comment by ChromaPIE** [comment]
Begging to support Shoulder Surfing Mod! @Snownee

**Comment by JustKenjiX** [comment]
I agree if possible a compatibility with the shoulder surfing (reloaded) mod would be an awesome addition.

**Comment by sepsol** [comment]
Maybe getting in touch with the dev of the other mod would be benefitial?

**Comment by The-furf-of-July** [comment]
Agreed, need this

**Comment by F-708** [comment]
It would be amazing with Shoulder Surfing compatibility
